### PR TITLE
add prettier to azure

### PR DESCRIPTION
**Pull request by jikim-msft**
Description
Board: https://dev.azure.com/fluidframework/internal/_workitems/edit/2118/

This PR allows running prettier in each sub-directory of release group and packages. Instead of adding the script to the root directory, developers can run npm format in whatever directory they are currently working on which will fix entire files accordingly to the prettier configuration.

There should be a separate PR to run prettier in Azure directory.

### CODE REVIEW

**Comment by alexvy86** [comment]
LGTM overall. I'd suggest we revert the changes to `azure/lerna-package-lock.json`, which are just around updating dependencies on other FF packgaes from 1.3.0 to 1.3.1. Doesn't seem relevant to the `prettier` changes, so better to let those updates happen only when they're really required. Letting @Josmithr have the last word on the PR approval since this the whole `prettier` effort is more his area.

**Review by Josmithr — APPROVED** [review]
Looks good! Thanks!

Be sure to remove the template boilerplate from the PR description before merging. It would also be a good idea to explain that you will be updating the formatting in a subsequent PR.

**Comment by jikim-msft** [comment]
> LGTM overall. I'd suggest we revert the changes to `azure/lerna-package-lock.json`, which are just around updating dependencies on other FF packgaes from 1.3.0 to 1.3.1. Doesn't seem relevant to the `prettier` changes, so better to let those updates happen only when they're really required. Letting @Josmithr have the last word on the PR approval since this the whole `prettier` effort is more his area.

Change applied. Thank you for the update!

**Comment by jikim-msft** [comment]
> Looks good! Thanks!
> 
> Be sure to remove the template boilerplate from the PR description before merging. It would also be a good idea to explain that you will be updating the formatting in a subsequent PR.

Also applied 😄. Thank you!

**Review by Josmithr — APPROVED** [review]

File: azure/packages/azure-local-service/package.json
Comment (ChumpChief) [inline azure/packages/azure-local-service/package.json:29]: This relative path is outside of the repo, maybe bad copy/paste?

File: azure/packages/test/scenario-runner/package.json
Comment (ChumpChief) [inline azure/packages/test/scenario-runner/package.json:29]: I don't think this .prettierignore exists?  I see one exists in /server, but not in /azure.

File: azure/packages/azure-local-service/package.json
Comment (Josmithr) [reply azure/packages/azure-local-service/package.json:29]: It's outside the azure mono-repo, but still in the fluidframework repo. There is an ignore file at the root of our repo. We may want to the ignore files into each mono-repo long-term, though, if we want to properly support each as its own isolated workspace.

Comment (ChumpChief) [reply azure/packages/azure-local-service/package.json:29]: Ah right - though typically configs and such in the root are implied to be specific to the client packages.

**Pull request merged**
